gpcontrib/arenadata_toolkit/expected/arenadata_toolkit_skew_test.out

```diff
@@ -7,7 +10,7 @@ SET search_path = arenadata_toolkit;
 -- m/(.*)size=\d+/
 -- s/(.*)size=\d+/$1size/
 -- end_matchsubs
--- function compares behaviour of pg_relation_size and adb_relation_storage_size functions for
+-- function compares behavior of pg_relation_size and adb_relation_storage_size functions for
```

Comment:
What the point of this change? Both are correct.

Reply:
My code editor lights the first variant as incorrect...